File: official/docs/java/apiKeys/RetrieveApiKeys.java
import com.easypost.EasyPost;

public class RetrieveApiKeys {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");
        
        ApiKeys parentKeys = ApiKeys.all();

        System.out.println(parentKeys);
    }
}

File: official/docs/csharp/api_keys/retrieve-api-keys.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using Newtonsoft.Json;
using EasyPost;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            string apiKey = Environment.GetEnvironmentVariable("EASYPOST_API_KEY")!;

            EasyPost.ClientManager.SetCurrent(apiKey);

            List<ApiKey> apiKeys = await ApiKey.All();

            Console.WriteLine(JsonConvert.SerializeObject(apiKeys, Formatting.Indented));
        }
    }
}
